pom.xml

```diff
@@ -260,6 +260,10 @@
                   <key>raml_files</key>
                   <value>${ramlfiles_path}</value>
                 </systemProperty>
+                <systemProperty>
+                  <key>schema_paths</key>
+                  <value>${jsonschema_paths}</value>
```

Comment:
Looks like definition of `jsonschema_paths` property is missed.

Reply:
Added the property.